lib/acquireRefs.js

```diff
@@ -9,17 +8,8 @@ var requireAccess = require('./requireAccess');
 
 module.exports = function(options, authScopes) {
   return requireAccess(options, authScopes).then(function() {
-    return new RSVP.Promise(function(resolve, reject) {
-      var firebaseRef = new Firebase(utils.addSubdomain(api.realtimeOrigin, 'firebase'));
-      firebaseRef.authWithCustomToken(options.metadataToken, function(err) {
-        if (err) {
-          return reject(new FirebaseError('Failed to authenticate to Firebase', {
-            original: err
-          }));
-        }
-        options.firebaseRef = firebaseRef;
-        resolve();
-      });
-    });
+    var firebaseRef = new Firebase(utils.addSubdomain(api.realtimeOrigin, 'firebase'));
+    options.firebaseRef = firebaseRef;
+    return RSVP.resolve();
```

Comment:
`acquireRefs` does two things:

1. Gets a token for the users' realtime database
2. Gets a downscoped token for the metabase

While (1) is no longer necessary, (2) still is and I suspect that if you tried to deploy to hosting on this branch it would fail. Let's keep this method around for use in deployment but no longer use it for db commands.